lib/signs/utilities/messages.ex

```diff
@@ -56,6 +80,39 @@ defmodule Signs.Utilities.Messages do
     end
   end
 
+  defp has_single_source_list?(%Signs.Realtime{source_config: {_}} = _sign), do: true
+  defp has_single_source_list?(_), do: false
+
+  defp get_paging_headway_or_alert_messages(
+         sign,
+         current_time,
+         alert_status,
+         line
+       ) do
+    if has_single_source_list?(sign) do
+      {nil, Content.Message.Empty.new()}
+    else
+      source_config =
+        if line == :top do
+          %Signs.Realtime{source_config: {top_line_source, _}} = sign
+          top_line_source
+        else
+          %Signs.Realtime{source_config: {_, bottom_line_source}} = sign
+          bottom_line_source
+        end
+
+      if alert_status == :none do
+        Signs.Utilities.Headways.get_paging_message(sign, source_config, current_time)
+      else
+        get_paging_alert_message(
+          alert_status,
+          sign.uses_shuttles,
+          Signs.Utilities.Headways.get_headway_destination(source_config)
+        )
+      end
+    end
+  end
```

Comment:
There's probably a cleaner way to do this, but wanted to get a PR up in the meantime. Open to suggestions!